src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -113,48 +132,140 @@ To chat with New Relic AI about log errors:
 
-Our AI uses a variety of inputs to improve its responses and always provides the best answer. For example, it uses inputs like our product documentation, a large library of example queries, and your telemetry data. But even though our AI scans your data as needed to provide context and help you build charts and understand response times, that data is _never stored_ by New Relic.
+For example:
 
-## Always learning 
+```
+Create a synthetic monitor to check the URL 'newrelic.com' every 10 minutes and name it: 'my awesome monitor'.
+```
 
-New Relic AI is becoming more accurate and responsive the more it's used. We want to build the AI experience you want. When chatting with our AI, please give its response a thumbs up or a thumbs down.  If you notice something that isn't right or that could be improved, let us know. Go to the user menu, click <DoNotTranslate>**Help**</DoNotTranslate>, then click <DoNotTranslate>**Give us feedback**</DoNotTranslate>.
+To create a ping check monitor with NRAI:
+
+    1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)** **All Entities**.
+    2. Click **Ask AI** in the top right-hand corner.
+    3. Tell AI to create a ping check monitor based on your parameters.
+    4. Click **Create monitor**.
+    5. If you want to see your settings or make any changes, select **Go to monitor**.
+    6. Select ** Detailed configuration ** if you want more advanced settings or access to other types of synthetic monitors.
+
+## Check the status of support tickets [#support]
+
+Ask NRAI for a summary of your support tickets to reduce the time to resolution. Revisit solutions and past issues to streamline management. You can also check to see if tickets remain open or can be closed as resolved. 
+
+Chat with AI and ask:
+
+```
+Summarize my support ticket(s).
+```
+```
+What is the status of my support ticket(s)?
+```
+
+## Demystify your dashboards [#dashboards]
+
+Dashboards are an incredibly useful tool, but if you're new to a specific entity or you're a product manager or executive, you might not understand all the nuances. New Relic AI can be your personal data analyst, automatically summarizing key information and uncovering hidden insights directly within the New Relic UI. 
+
+    1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Dashboards**</DoNotTranslate>
+    2. Click **Ask AI** in the top right-hand corner.
+
+Even basic questions can provide detailed summaries that can help clarify complicated data. Try asking: 
+
+```
+What is on this dashboard?
+```
+
+```
+Summarize this dashboard for me. 
+```
+
+## Enable NRAI [#enable]
```

Comment:
@homelessbirds it seems like the horse is already out of the barn. You've already been talking about this, so it seems like an afterthought here. How about moving this closer to the beginning?

Reply:
Great idea. Will do.